integration-tests/freemarker/src/main/resources/application.properties

```diff
@@ -0,0 +1,20 @@
+## ---------------------------------------------------------------------------
+## Licensed to the Apache Software Foundation (ASF) under one or more
+## contributor license agreements.  See the NOTICE file distributed with
+## this work for additional information regarding copyright ownership.
+## The ASF licenses this file to You under the Apache License, Version 2.0
+## (the "License"); you may not use this file except in compliance with
+## the License.  You may obtain a copy of the License at
+##
+##      http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+#
+# Quarkus
+#
+quarkus.camel.freemarker.locations = org/apache/camel/component/freemarker
```

Comment:
Given that `org/apache/camel/component/freemarker` will be scanned recursivelly, I wonder whether all the class files are going to get included as native image resources? I would be interesting to compare the size of the native image using this locations value and using a new path that does not contain any class files.

Reply:
Only files with the extension `.ftl` will be included. You can see there is a filter for that [here](https://github.com/carlosthe19916/camel-quarkus/blob/7d45a783e5b27d341b6a07df49b34087f636dad3/extensions/freemarker/deployment/src/main/java/org/apache/camel/quarkus/component/freemarker/deployment/FreemarkerNativeImageProcessor.java#L119)... I've also created 2 native images, the first one using `org/apache/camel/component/freemarker` and the second one using `myfreemarkerfolder` and the sizes are exactly the same (45.8 MB)

Reply:
> Only files with the extension `.ftl` will be included.

Thanks for the info, I have not noticed that and I must say I do not think we should be selecting only `*.ftl` files. I commonly used other or no extensions for FreeMarker template files in the past. I think many people do that because because of syntax highlighting in editors. There is even a couple of examples in this repo: https://github.com/apache/camel-quarkus/tree/master/tooling/create-extension-templates

I think we should either include whole subtrees or we should have path includes and excludes like we have in https://github.com/apache/camel-quarkus/blob/master/extensions/core/runtime/src/main/java/org/apache/camel/quarkus/core/CamelConfig.java#L74-L100 WDYT?

Reply:
Good point @ppalaga . Files with different extensions should still work. I agree and we should include the whole subtree but just to avoid adding files we don't want to add we can use `includePatterns` and `excludePatterns` as you suggested. Let me add it

Reply:
@ppalaga sorry for being so slow in this. I've added `includePatterns` and `excludePatterns`